### DISCUSSION TITLE
Welcome to Forem Discussions 🥳

### DISCUSSION THREAD

**Discussion by cmgorton** [forem/forem#15313] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Add feature requests. If you see a feature request you would like implemented please upvote it or leave a comment. 
- Ask questions about contributing to Forem. 
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ludwiczakpawel** [comment]
FIRST!!!!!111!!111!one11!

**Reply by cmgorton** [reply]
@ludwiczakpawel I got my eye on you 👁
